src/components/UsersTableComponent.js

```diff
@@ -42,18 +43,18 @@ const columns = [
     formatter: (rowContent, row) => {
       return (
         <div style={{ display: "flex", justifyContent: "space-around" }}>
-          <Link to={"edit/"+row.id}>
-          <Button color="primary">
-            <FontAwesomeIcon icon={faEdit} />
-          </Button>
+          <Link to={"edit/" + row.id}>
+            <Button color="primary">
+              <FontAwesomeIcon icon={faEdit} />
+            </Button>
           </Link>
           <Button color="danger">
             <FontAwesomeIcon icon={faTrash} />
           </Button>
-          <Link to={"detail/"+row.id}>
-          <Button color="secondary">
-            <FontAwesomeIcon icon={faInfo} />
-          </Button>
+          <Link to={"detail/" + row.id}>
```

Comment:
use string template literal instead @azukhrufy

Reply:
ok got it, thank you